src/components/forms/FormGroup/styles.js

```diff
@@ -0,0 +1,37 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export default makeStyles(
+  theme => ({
+    'form-group-header': {
```

Comment:
any reason for the change to hyphenated string keys here?
i.e. why not `formGroupHeader` so that it can be accessed with `styles.formGroupHeader`?

Reply:
Nope, other than personal preference and css conventions. I personally find it easier to read, but I'm not picky!